src/containers/paper-canvas.jsx

```diff
@@ -45,6 +50,19 @@ class PaperCanvas extends React.Component {
     }
     componentWillUnmount () {
         paper.remove();
+        document.removeEventListener('keydown', this.handleKeyDown);
+    }
+    handleKeyDown (event) {
+        if (event.target instanceof HTMLInputElement) {
+            // Ignore delete if a text input field is focused
+            return;
+        }
+        // Backspace, delete
+        if (event.key.toLowerCase() === 'delete' || event.key.toLowerCase() === 'backspace') {
```

Comment:
I don't think there is any capitalization variation with `event.key`, should just be `Delete` or `Backspace`

Reply:
Done